src/components/github-htmlview.component.js

```diff
@@ -48,6 +48,10 @@ const styles = {
   code: codeStyle,
   span: textStyle,
   p: { ...textStyle, margin: 0, padding: 0 },
+  sub:{ 
+    ...textStyle,
+    fontSize: normalize(9),
+  },
```

Comment:
and a new screenshot after these changes so we can verify it's still working as expected 🙏

Reply:
updated thanks.